src/automata_pccs/AutomataFmspcTcbDao.sol

```diff
@@ -17,4 +17,31 @@ contract AutomataFmspcTcbDao is AutomataDaoBase, FmspcTcbDao {
     {
         data = super._onFetchDataFromResolver(key, hash);
     }
+
+    /// @dev submit tcb issue date timestamp and evaluation data number as a separate attestation
+    /// TEMP: it is not the most efficient approach, since it's storing duplicate data
+    /// @dev if i could extract the required info directly from the attestation,
+    /// this method will no longer be needed
+    /// @dev this is a good TODO for future optimization
+    function _cacheTcbInfoIssueEvaluation(bytes32 tcbKey, uint64 issueDateTimestamp, uint32 evaluationDataNumber) internal override {
```

Comment:
Why do you use the term cache?

Reply:
Maybe "caching" isn't the appropriate term for this action. But the goal is to store a copy of the issued timestamp and evaluation number, to reduce the read amount for validity check.

I can just rename it as `_storeTcbInfoIssueEvaluation`

Reply:
[9661dc2](https://github.com/automata-network/automata-on-chain-pccs/pull/15/commits/9661dc20b2c228cb33480ca5538485f243e911db)